components/body/index.styl

```diff
@@ -154,6 +159,11 @@
       color gray
       font-size 15px
 
+  a.artwork
```

Comment:
Repeated style here? I would try to pull the styles that aren't specific to breakpoints out and just declare them outside of that context. That way you don't have to define them twice.